src/main/java/uk/gov/companieshouse/acsp/controllers/AcspController.java

```diff
@@ -92,7 +92,7 @@ public ResponseEntity<Object> checkHasApplication(@PathVariable("acsp_applicatio
         return acspService.getAcspApplicationStatus(acspId, requestId);
     }
 
-    @DeleteMapping("/acsp-api/user/{acsp_application_id}/application")
+    @DeleteMapping("/transactions/{" + TRANSACTION_ID_KEY + "}/authorised-corporate-service-provider-applications/{acsp_application_id}")
```

Comment:
Please dont update the existing endpoint... 
create a new one as discussed before. We are not ready to make a change in acsp-web related to this